core/src/main/java/com/linecorp/armeria/client/pool/DefaultKeyedChannelPool.java

```diff
@@ -74,6 +84,7 @@ public DefaultKeyedChannelPool(EventLoop eventLoop, Function<K, Future<Channel>>
 
         pool = new ConcurrentHashMap<>();
```

Comment:
We can make this a `HashMap` now that `pool` is only accessed from `eventLoop`. `close()` was the only place that accesses `pool` out of `eventLoop`. I may have misread the code, so would you mind double-checking?

Reply:
Sweet, thanks for noticing!